### PR TITLE
Build OTP 27

### PR DESCRIPTION
**Pull request by rodrigues**
Hi there 👋

Possibly still too early for this?

Thanks!

### CODE REVIEW

File: lib/bob/job/build_elixir.ex [hunk 1]
```diff
@@ -16,7 +16,7 @@ defmodule Bob.Job.BuildElixir do
     version = ref_to_version(ref_name)
 
     cond do
-      version_gte(version, "1.15.0-0") -> ["24.3", "25.3", "26.0"]
+      version_gte(version, "1.15.0-0") -> ["24.3", "25.3", "26.0", "27.0"]
```
Comment (josevalim) [inline lib/bob/job/build_elixir.ex:19]: Only Elixir v1.17 will work on OTP 27.

**Comment by rodrigues** [comment]
@ericmj sure thing!

**Comment by rodrigues** [comment]
reopen

File: lib/bob/job/build_elixir.ex [hunk 2]
```diff
@@ -16,6 +16,7 @@ defmodule Bob.Job.BuildElixir do
     version = ref_to_version(ref_name)
 
     cond do
+      version_gte(version, "1.17.0-0") -> ["25.3", "26.0", "27.0"]
```
[the remark this answers is not in the document]
Comment (ericmj) [reply lib/bob/job/build_elixir.ex:19]: This should work

**Comment by ericmj** [comment]
Thank you!

**Comment by ericmj** [comment]
No, they are not compatible https://hexdocs.pm/elixir/main/compatibility-and-deprecations.html#between-elixir-and-erlang-otp.

Elixir 1.17 RC will be released soon which is compatible with OTP 27.

**Comment by rodrigues** [comment]
@ericmj elixir `1.17.0-rc.0` is not building with latest OTP 27, it picked just `27.0-rc1`, something we should change here?

thanks!

Screenshot 2024-05-24 at 18 30 30

**Comment by ericmj** [comment]
The docker builds are derived from DockerChecker, not this job.

**Comment by ericmj** [comment]
@rodrigues Bad sorting caused the dedup check to work incorrectly. We are building OTP 27.0 now.

https://github.com/hexpm/bob/commit/28d2bacf5470e12756530f5e1d72eeb07ee48d99

**Pull request merged**